Charith/example-1/index.html

```diff
@@ -7,92 +7,198 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
     <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <link rel="stylesheet" href="./assets/css">
-
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="./assets/css/main.css">
 </head>
 <body>
 <!-- This snippet uses Font Awesome 5 Free as a dependency. You can download it at fontawesome.io! -->
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
 
 <section class="pricing py-5 bg-white">
     <div class="container">
         <div class="row">
             <!-- Free Tier -->
             <div class="col-lg-4 bg-light">
                 <div class="card mb-5 mb-lg-0">
-                    <div class="card-body">
+                    <div class="card-body img-thumbnail">
                         <div class="row">
                             <div class="col-md-8 mr-auto text-left">
```

Comment:
Why did you use 'text-left' with 'mr-auto'? Only one of those classes are enough